controllers/cloud.redhat.com/providers/kafka/appinterface.go

```diff
@@ -92,11 +95,12 @@ func validateBrokerService(ctx context.Context, cl client.Client, nn types.Names
 	err := cl.Get(ctx, nn, &svc)
 
 	if err != nil {
-		errors.LogError(ctx, "kafka", errors.New("Cannot find kafka bootstrap service"))
-		qualifiedName := fmt.Sprintf("%s:%s", nn.Namespace, nn.Name)
-		return &errors.MissingDependencies{
-			MissingDeps: map[string][]string{"kafka": {qualifiedName}},
-		}
+		missingDeps := errors.MakeMissingDependencies(errors.MissingDependency{
+			Source:  "kafka",
+			App:     nn.Name,
+			Details: fmt.Sprintf("No kafka bootstrap service for app '%s' found in namespace '%s'", nn.Name, nn.Namespace),
```

Comment:
Since we tried to 'Get' a `core.Service` resource with the client up above...

```suggestion
			Details: fmt.Sprintf("No Service named '%s' found in namespace '%s'", nn.Name, nn.Namespace),
```

Reply:
Oh, lol - this invalidates my change above. Changed to this :)